ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/Button.kt

```diff
@@ -40,24 +50,30 @@ fun PrimaryButton(
         horizontal = ZcashTheme.dimens.spacingDefault,
         vertical = ZcashTheme.dimens.spacingSmall
     ),
-    enabled: Boolean = true
+    enabled: Boolean = true,
+    buttonColor: Color = MaterialTheme.colorScheme.primary,
+    textColor: Color = MaterialTheme.colorScheme.onPrimary,
 ) {
     Button(
         shape = RectangleShape,
+        enabled= enabled,
+        modifier = modifier
+            .padding(outerPaddingValues)
+            .shadow(
+                Color.Black,
+                borderRadius = 0.dp,
+                offsetX = 20.dp,
+                offsetY = 20.dp,
+                spread = 10.dp,
+                blurRadius = 0.dp,
+                stroke = textColor != Color.Black,
+            )
+            .defaultMinSize(230.dp, 50.dp)
```

Comment:
Should we move these sizes to the `Dimens` so we can customize them once we decide to support different screen sizes, orientations, or directions?